src/Standards/Squiz/Sniffs/PHP/EmbeddedPhpSniff.php

```diff
@@ -379,8 +379,11 @@ private function validateInlineEmbeddedPhp($phpcsFile, $stackPtr, $closeTag)
         }
 
         // Check that there is one, and only one space at the start of the statement.
-        $leadingSpace = 0;
-        if ($tokens[$stackPtr]['code'] === T_OPEN_TAG) {
+        $leadingSpace  = 0;
+        $isLongOpenTag = $tokens[$stackPtr]['code'] === T_OPEN_TAG
+                         && stripos($tokens[$stackPtr]['content'], '<?php') === 0;
+
+        if ($isLongOpenTag === true) {
             // The long open tag token in a single line tag set always contains a single space after it.
             $leadingSpace = 1;
         }
```

Comment:
What's with the non-standard indentation of line 384 here ?

(Good call on setting a variable though, what with the condition being needed again a little further down)

```suggestion
        $leadingSpace  = 0;
        $isLongOpenTag = false;
        if ($tokens[$stackPtr]['code'] === T_OPEN_TAG
            && stripos($tokens[$stackPtr]['content'], '<?php') === 0
        ) {
            // The long open tag token in a single line tag set always contains a single space after it.
            $leadingSpace  = 1;
            $isLongOpenTag = true;
        }
```

Reply:
> What's with the non-standard indentation of line 384 here ?

I don't know what would be the standard way to indent this line according to the PHPCS coding standard.

Reply:
When in doubt and PHPCS itself does not give guidance when run over itself, always indent with one tab-stop (four spaces).